app/assets/stylesheets/responsive/custom.scss

```diff
@@ -1917,4 +1917,10 @@ $color_asktheeu_dark_green: #C4DDB9;
     display: none;
     visibility: hidden;
   }
-}
\ No newline at end of file
+}
+
+.act-link-icon {background-size: 144px 16px;}
```

Comment:
We usually try to keep CSS declarations on separate lines, even when there’s only one in the ruleset. That’d make this ruleset look like:

```
.act-link-icon {
  background-size: 144px 16px;
}
```